backend/routes/auth.js

```diff
@@ -24,6 +24,8 @@ auth.post('/login', parsers.json, async ctx => {
             ctx.status = 200;
             response.result = 'success';
             response.username = user.username;
+            response.is_admin = user.is_admin
```

Comment:
I didn't think of this before, but can we have a little more sophisticated  format `response`? I think it's more elegant:
response = {
  status: 'success' .
  data: { ..user fields: id, username, admin.. }
  
This was we'll be able to handle the user object as one and also I like `status` and `data` better than `result`

Thanks!